sensor/common/compliance/node_matcher.go

```diff
@@ -0,0 +1,38 @@
+package compliance
+
+import (
+	"fmt"
+
+	"github.com/stackrox/rox/sensor/common/store"
+)
+
+// nodeStore provides functionality to get nodes
+type nodeStore interface {
+	AddOrUpdateNode(node *store.NodeWrap) bool
```

Comment:
This doesn't seem to be used by production code, only for tests. Why does it need to be part of the interface?